### ISSUE TITLE
Population data presentation

### ISSUE THREAD

**Issue by FredrikAugust** [IT2901-SINTEF01/frontend#112]
### Description
As of now, the population data visualisation doesn't work.

Your task is to figure out how it should work, and make it work.

Example solution:
- The user selects one municipality from a list of municipalities on the visualisation selection page. This requires you to perform a graphql query on the visualisation selection page to retrieve the available municipalities, as well as the years available.
- You would then need to include the selections in the global store, and map it to an input to the line/threshold charts which will then be rendered on the front page

### Subtasks
- [x] Come up with a solution
- [x] Implement said solution

**Comment by FredrikAugust** [comment]
Updated in #113 so the data source _works_, but not fully.

**Comment by fredrikbw** [comment]
## Proposed solution:

#### Selection of municipality:
* Dropdown menu similar to when selecting visualization type
* All options in this menu comes from the following query 
```
query {
	populationsInNorway {
		municipalitiesWithKeys
	}
}
```
* You should only be able to gather data from one municipality at a time

#### Selection of years:
_I suppose it wont be necessary to query for available years, as this should be the same each municipality? (1986 - current_year)_
* With a slider or start/end input field, easy to implement, but won't be able to skip years (only see 2010, 2011, 2013 for instance)
* Should one have the ability to skip years?

The selected years , together with the selected municipality will then be used in a query. For instance pass them as params to the [populationInNorway.ts](https://github.com/IT2901-SINTEF01/frontend/blob/master/src/queries/populationInNorway.ts) query.

**Comment by FredrikAugust** [comment]
Fixed in #146
